src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule731selfexplanatory/InputJavadocMethodAndMissingJavadocMethod.java

```diff
@@ -138,6 +138,32 @@ String foo91() {
   public String foo92() {
     return "Fooooo" + "ooo" + "ooooooo" + "ooooo" + "ooo";
   }
+
+  int xyz = 99;
+
+  public int getXyz() {
+    return xyz;
+  }
+
+  public void setXyz(int xyz) {
+    this.xyz = xyz;
+  }
+
+  int abc = 101;
+
+  // violation below 'Missing a Javadoc comment.'
+  public int getAbc() { // this will not work
```

Comment:
please remove this comment it is confusing

Reply:
done